addons/photo_collage/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_photo_collage,photo_collage.photo_collage,model_photo_collage,base.group_user,1,0,0,0
```

Comment:
`base.group_user` user would prevent logged out users from viewing those objects. Leave the "group_id" field empty.